package.py

```diff
@@ -3,7 +3,7 @@
 name = 'shotgun_api3'
 
 _shotgunSoftwareVersion = '3.1.1'
-_rdoVersion = '1.1.0'
+_rdoVersion = '1.1.1'
```

Comment:
Should this be 1.1.2 since #21 declares 1.1.1?

Reply:
Both could go at the same time, so no.